src/main/antlr/Graphql.g4

```diff
@@ -1,5 +1,5 @@
 grammar Graphql;
-import GraphqlSDL, GraphqlOperation;
```

Comment:
this is needed more for IDEA and its "run ANLTR rule" support but it does no harm

Reply:
I don't understand: we need to import the rules we wanna use ... why is this only needed for IDEA?

Reply:
IDEA needs this to be able to use the "Run Rule" plugin that helps you run the grammar interactively.  Without this you can do that.  It does not work out the transitive dependency

However at actual generation time, it works out the transitive dependency ok.

So its harmless but really helps in development that you can run the rule interactively in IDEA.

Check it out.  Its a cool IDEA feature

![image](https://user-images.githubusercontent.com/1661769/36411008-c8ac9ce0-1667-11e8-9aea-aacd5456339a.png)

Reply:
ah ok .. the only consequence is that we actually generate code for every rule in graphqlCommon, even if it is not really used. But on the other side there should be not unused rule, so 👍